actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -106,7 +106,7 @@ def _compute_redirect_to_location(request, options) #:nodoc:
       # See https://tools.ietf.org/html/rfc3986#section-3.1
       # The protocol relative scheme starts with a double slash "//".
       when /\A([a-z][a-z\d\-+.]*:|\/\/).*/i
-        options
+        options.to_str
```

Comment:
Why `#to_str` and not `to_s` ? Am I missing something?

Reply:
We're relying on `#to_str` result for regexp match, so I've changed it to `#to_str`. The sane object will have the same result for `#to_str` & `#to_s`, but who knows. 
Anyway, I don't have a strong feeling about it, so will change if needed.